frontend/package.json

```diff
@@ -1,259 +1,26 @@
 {
-  "name": "streamlit-browser",
-  "version": "1.23.1",
+  "name": "streamlit",
+  "version": "1.23.0",
```

Comment:
See earlier note re: `update_version.py`. Also, let's make sure that all these version numbers match the current streamlit version (1.23.1)